controllers/administration.js

```diff
@@ -3147,6 +3147,19 @@ router.use(
 	},
 );
 
+router.use('/administration/policies/:id', async (req, res, next) => {
+	permissionsHelper.permissionsChecker('ADMIN_VIEW');
```

Comment:
Why not execute as middleware operation?

Reply:
Was implemented like other calls in administration.js